# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/avm/noteedit/hotfixes.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./noteedit/timesigDia.ui.orig	2005-10-31 18:39:26.000000000 +0100
+++ ./noteedit/timesigDia.ui	2005-10-31 18:39:30.000000000 +0100
@@ -357,7 +357,5 @@
 </slots>
 <layoutdefaults spacing="6" margin="11"/>
 <includehints>
-    <includehint>nscaleedit.h</includehint>
-    <includehint>nscaleedit.h</includehint>
 </includehints>
 </UI>
--- ./noteedit/staffProps.ui.orig	2005-10-31 18:39:09.000000000 +0100
+++ ./noteedit/staffProps.ui	2005-10-31 18:39:15.000000000 +0100
@@ -543,13 +543,5 @@
 </slots>
 <layoutdefaults spacing="6" margin="11"/>
 <includehints>
-    <includehint>nscaleedit.h</includehint>
-    <includehint>nscaleedit.h</includehint>
-    <includehint>nscaleedit.h</includehint>
-    <includehint>nscaleedit.h</includehint>
-    <includehint>nscaleedit.h</includehint>
-    <includehint>nscaleedit.h</includehint>
-    <includehint>nscaleedit.h</includehint>
-    <includehint>nscaleedit.h</includehint>
 </includehints>
 </UI>
